go-packages/meep-metric-store/events.go

```diff
@@ -55,7 +55,11 @@ func (ms *MetricStore) GetEventMetric(eventType string, duration string, count i
 	}
 
 	// Get Traffic metrics
-	tags := map[string]string{EvMetType: eventType}
+	//tags := map[string]string{EvMetType: eventType}
```

Comment:
Previous implementation assumed an event type was specified. This is the first time we need to get all the event types. Does the API meet the requirements? Should the returned event metrics include event type information?

Reply:
seems flexible enough. Removing it allowed querying every types, like a wildcard. I believe this is good enough here. It also allows for unknown type to be returned. I don't think changes are needed